tests/gordo_components/model/test_raw_keras.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import yaml
+import tensorflow as tf
+import numpy as np
+
+from sklearn.pipeline import Pipeline
+
+from gordo_components import serializer
+from gordo_components.model.models import KerasRawModel
+
+
+@pytest.mark.parametrize(
+    "spec_str",
+    (
+        """
+        compile:
+            loss: mse
+            optimizer: sgd
+        spec:
+            tensorflow.keras.models.Sequential:
+                layers:
+                    - tensorflow.keras.layers.Dense: 
+                        units: 10
+                    - tensorflow.keras.layers.Dense:
+                        units: 32
+                        kernel_regularizer:
+                            tensorflow.keras.regularizers.L1L2:
+                                l1: 0.2
+                    - tensorflow.keras.layers.Dense:
+                        units: 1
+    """,
+    ),
+)
+def test_raw_keras_basic(spec_str: str):
+    """Can load a keras.Sequential model from a config/yaml definition(s)"""
```

Comment:
Perhaps use new lines for the quotes in docstring, as we do other places 
"""
Can load a keras.Sequential model from a config/yaml definition(s)
"""